consensus/chain_reader.go

```diff
@@ -0,0 +1,116 @@
+package consensus
```

Comment:
try use `consensuschain.NewReader` and remove this file

Reply:
File removed, was actually unused